js/screen/game.js

```diff
@@ -74,6 +77,29 @@ function create () {
     checkTwoPlayers();
   };
 
+  /**
+   * Checks if two players are connected!
+   */
+  function checkTwoPlayers() {
+    const active_players = airconsole.getActivePlayerDeviceIds();
+
+    // Only update if the game didn't have active players.
+    if (active_players.length == 0) {
```

Comment:
How does this work?

IMO, we should make it so that if two players are connected, we initiate a STATE.STARTED
If 0 or 1 player are connected, we should be in STATE.WAITING.
the only reason we should get to STATE.END is after there are no more songs to play (see code in sendNewSong).